src/d3d11/d3d11_context.cpp

```diff
@@ -2858,9 +2858,8 @@ namespace dxvk {
           pViewports[i].MaxDepth = 0.0f;
         }
       }
-    }
-    
-    *pNumViewports = m_state.rs.numViewports;
+    } else
+      *pNumViewports = m_state.rs.numViewports;
```

Comment:
`else if (pNumViewports != nullptr)` perhaps? Not in our best interests to crash from user data.

Reply:
Windows D3D11 also just crashes when you pass null for that.